LICENCE

```diff
@@ -44,3 +70,51 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+====
+
+Apache License 2.0
+
+Copyright Typescript
```

Comment:
Typescript paketin lisenssissä ei sanota sitä kenelle Copyright kuuluu niin laitoin nyt sitten vaan näin👀 Sehän on Microsoftin toisaalta, että sen voisi melkein tuohon laittaa🤔